addons/l10n_tr/data/account.group.template.csv

```diff
@@ -0,0 +1,65 @@
+"tr_group_69","tr_group_6",69,69,"DÖNEM NET KÂRI (ZARARI)","l10n_tr.chart_template",
+"tr_group_7",,7,7,"MALİYET HESAPLARI","l10n_tr.chart_template","Maliyet hesapları, mal ve hizmetlerin planlanan biçim ve niteliğe getirilmesi için yapılan giderlerin toplandığı ve maliyet unsurlarına dönüştürülerek izlendiği hesaplardır."
+"tr_group_70","tr_group_7",70,70,"MALİYET MUHASEBESİ BAĞLANTI HESAPLARI","l10n_tr.chart_template","Bu grupda yer alan hesaplar genel muhasebe ile maliyet muhasebesinin birbirinden ayrı olarak yürütülmesinin istenmesi durumunda, genel muhasebe ile maliyet muhasebesi arasında bağlantıların kurulmasını sağlamak amacıyla kullanılır."
+"tr_group_71","tr_group_7",71,71,"DİREKT İLKMADDE VE MALZEME GİDERLERİ","l10n_tr.chart_template","Bu grupta yer alan hesaplar, üretilen mamülün bünyesine ana madde olarak katılan ve hangi mamülden ne kadar tüketildiği, ekonomik bakımdan ayrı olarak izlenebilen ilkmadde ve malzemelerle ilgili gider, yansıtma ve fark hesaplarını kapsar."
+"tr_group_72","tr_group_7",72,72,"DİREKT İŞÇİLİK GİDERLERİ","l10n_tr.chart_template","Bu grup bir mal veya hizmetin üretim maliyetine doğrudan doğruya verilebilen işçilik giderleri ile ilgili gider, yansıtma ve fark hesaplarından oluşur."
+"tr_group_73","tr_group_7",73,73,"GENEL ÜRETİM GİDERLERİ","l10n_tr.chart_template","Bu grup işletmenin üretimi ve bu üretime bağlı hizmetler için yapılan direkt ilk madde ve malzeme ve direkt işçilik dışında kalan giderlerle ilgili gider, yansıtma ve fark hesaplarından oluşur."
+"tr_group_74","tr_group_7",74,74,"HİZMET ÜRETİM MALİYETİ","l10n_tr.chart_template","Bu grup hizmet işletmeleri ile ilgilidir. Hizmet işletmelerinde 71, 72 ve 73 gruplarda yer alan hesaplar kullanılmaz. Bu hesaplar yerine bu gruptaki hesaplar kullanılır. Hizmet işletmelerinin üretim maliyetlerinin izlenmesinde kullanılan hesaplar bu grupta yeralır."
+"tr_group_75","tr_group_7",75,75,"ARAŞTIRMA VE GELİŞTİRME GİDERLERİ","l10n_tr.chart_template","Bu grup araştırma ve geliştirme faaliyetleri ile ilgili gider, yansıtma ve fark hesaplarından oluşur."
+"tr_group_76","tr_group_7",76,76,"PAZARLAMA SATIŞ VE DAĞITIM GİDERLERİ","l10n_tr.chart_template","Bu grup araştırma ve geliştirme faaliyetleri ile ilgili gider, yansıtma ve fark hesaplarından oluşur."
+"tr_group_77","tr_group_7",77,77,"GENEL YÖNETİM GİDERLERİ","l10n_tr.chart_template","Bu grup yönetim fonksiyonu ile ilgili gider, yansıtma ve fark hesaplarından oluşur."
+"tr_group_78","tr_group_7",78,78,"FİNANSMAN GİDERLERİ","l10n_tr.chart_template","Bu grupta işletmenin borçlandığı tutarlarla ilgili olarak katlanılan ve varlıkların maliyetine eklenmemiş bulunan faiz ve benzeri giderlerle, yansıtma ve fark hesapları yer alır."
+"tr_group_79","tr_group_7",79,79,"GİDER ÇEŞİTLERİ","l10n_tr.chart_template",
+"tr_group_9",,9,9,"NAZIM HESAPLAR","l10n_tr.chart_template",
```

Comment:
It's not a big deal but are the caps really mandatory?

Reply:
I don't think It is not mandatory but it is quite common in Turkey. Almost anything other than sentences are uppercase.